.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: test
+
+on:
+  workflow_call:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: yarn
```

Comment:
Use `yarn --frozen-lockfile` please.

Reply:
Added in [`a980e68` (#493)](https://github.com/facebook/jscodeshift/pull/493/commits/a980e68108633290d2d95e320c19586c2c497e48)

Tested on fork:
* 12.x https://github.com/trivikr/jscodeshift/runs/5699022264?check_suite_focus=true
* 14.x https://github.com/trivikr/jscodeshift/runs/5699022321?check_suite_focus=true
* 16.x https://github.com/trivikr/jscodeshift/runs/5699022359?check_suite_focus=true